code/game/machinery/electrolyzer.dm

```diff
@@ -0,0 +1,181 @@
+#define ELECTROLYZER_MODE_STANDBY	"standby"
+#define ELECTROLYZER_MODE_WORKING	"working"
+
+/obj/machinery/electrolyzer
```

Comment:
why isn't this an atmos machine?

Reply:
Well I've tried to make it an atmos machine, but no matter what I did it would never work or compile